homeassistant/components/iss/binary_sensor.py

```diff
@@ -23,88 +25,60 @@
 DEFAULT_NAME = "ISS"
 DEFAULT_DEVICE_CLASS = "visible"
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
-
 
 async def async_setup_entry(
     hass: HomeAssistant,
     entry: ConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the sensor platform."""
+    coordinator: DataUpdateCoordinator[IssData] = hass.data[DOMAIN]
+
     name = entry.title
     show_on_map = entry.options.get(CONF_SHOW_ON_MAP, False)
 
-    try:
-        iss_data = IssData(hass.config.latitude, hass.config.longitude)
-        await hass.async_add_executor_job(iss_data.update)
-    except HTTPError as error:
-        _LOGGER.error(error)
-        return
-
-    async_add_entities([IssBinarySensor(iss_data, name, show_on_map)], True)
+    async_add_entities([IssBinarySensor(coordinator, name, show_on_map)], True)
```

Comment:
We don't need to refresh the coordinator data again.
```suggestion
    async_add_entities([IssBinarySensor(coordinator, name, show_on_map)])
```